docs/text_handling.md

```diff
@@ -4,6 +4,19 @@ OpenVINO Model Server can now greatly simplify writing the applications with Nat
 
 We addressed both the situation when the original model requires tokens on input or output and there is added support for models with embedded tokenization layer. Below are demonstrated use cases with a simple client application sending and receiving text in a string format. Whole complexity of the text conversion is fully delegated to the remote serving endpoint.
 
+## Accepting strings by serving Python script
+
+When the model server is configured to serve python script (via MediaPipe Graph with PythonExecutorCalculator), it is possible to communicate via arbitrary format including string or list of strings. Refer to full [Python execution documentation](python_support/reference.md) or end-to-end [LLM text generation demo](../demos/python_demos/llm_text_generation/README.md).
+
+## Custom CPU Extension (Custom Operations) for tokenization via OpenVINO model
+
+Some AI model training frameworks support the layers accepting the string format on the input or output. They include the layers performing the tokenization operations inside the neural network.
+While core OpenVINO does not contain the layers accepting string data type, it is possible to extend it with a CPU extension.
+Model Server includes a built-in extension for wide list of custom [tokenizers and detokenizers](https://github.com/openvinotoolkit/openvino_contrib/tree/master/modules/custom_operations/user_ie_extensions/tokenizer) layers.
+The extension enables accepting OpenVINO tensors of type `ov::element::Type_t::string` and performing tokenization. The server creates string tensor out of KServe/TensorflowServing API strings automatically.
+
+A demonstration of such use case is in the MUSE model which can be imported directly but the models server. The client can send the text data without any preprocessing and take advantage of much faster execution time.
+Check the [MUSE demo](../demos/universal-sentence-encoder/README.md).
 
 ## DAG pipeline to delegate tokenization to the server
```

Comment:
```suggestion
## DAG pipeline to delegate tokenization to the server (deprecated)
```

Reply:
Are we going to remove 2d u8? This will remove any support for sending strings to custom nodes.

Reply:
Will invalidate blingfire-based custom node we release.